FluidNC/src/Motors/TrinamicBase.cpp

```diff
@@ -120,6 +120,19 @@ namespace MotorDrivers {
         }
     }
 
+    bool TrinamicBase::reportCommsCheck(bool result) {
```

Comment:
In general, please conform to the codebase's existing formatting conventions.  The approved way to do that is to run clang-format on the files that have changed, with the .clang-format ruleset that is at the top of the tree.

In this case, a better expression would be:
```
log_info(axisName() << " communications check " << (result ? "passed" : "failed"));
return result.
```
But it would be better still to have report CommsCheck() return void instead of propagating the bool through the function.  I realize that the propagation leads to a tighter expression in ::test(), but at the expense of the reader having to wonder how reportCommsCheck might possibly compute a new value, when in fact it does not change the value.  With the void return, ::test() would look like:
```
bool okay = ifcnt_after != ifcnt_before;
reportCommsCheck(okay);
return okay;
```
but wait, I'm still not done nit-picking.  The check would be more precise as follows:
```
bool okay = ((ifcnt_before +1) & 0xff) == ifcnt_after;
```
and one final thing:  We generally avoid displaying "success" messages in startup logs because too much verbiage makes it hard to find the interesting stuff.  The existing messages typically contain extra information like pin numbers, setting values, etc.  So it would be better to issue a message only on failure.  reportCommsCheck(bool) would thus become reportCommsFailure(void).